.github/workflows/release.yml

```diff
@@ -65,7 +65,12 @@ jobs:
         run: |
           mkdir -p /tmp/artifact/
           cp "target/${{ matrix.platform.target }}/release/${{ matrix.platform.bin }}" "/tmp/artifact/${{ matrix.platform.file-name }}"
+      - name: calculate sha256sum
+        run: |
+          sha256sum /tmp/artifact/${{ matrix.platform.file-name }} | cut -d " " -f 1 >> /tmp/artifact/${{ matrix.platform.file-name }}.sha256
```

Comment:
Why are you doing this: `cut -d " " -f 1`?

Reply:
cuz the normal output of `sha256sum` is like this:

```
aac93719f2ea7723200152068b3e4de14c6921fc46e69a4afe035c3f2fb40c17  am.exe
```

and that command removes the filename after so its just the checksum:

```
aac93719f2ea7723200152068b3e4de14c6921fc46e69a4afe035c3f2fb40c17
```

Reply:
I think we should keep it. That way we can just use `sha256sum -c am....sha256`

Reply:
oh i didnt know about that lol thats so useful